.github/workflows/pull_request.yml

```diff
@@ -46,9 +52,52 @@ jobs:
       - uses: actions/cache@v1
         with:
           path: /tmp/release-snacks-case-studies/
-          key: release-snacks-case-studies
+          key: release-snacks-case-studies-${{ runner.os }}-${{ hashFiles('**/flyte_tests_manifest.json') }}-${{ github.sha }}
       - name: Generate Serialize proto
         env:
           REGISTRY: "ghcr.io/flyteorg"
         run: |
-          make -C cookbook/case_studies serialize
\ No newline at end of file
+          make -C cookbook/case_studies serialize
+          mkdir -p cookbook/release-snacks /tmp/release-snacks-case-studies/
+          make -C cookbook archive
+          cp -r cookbook/release-snacks/* /tmp/release-snacks-case-studies/
+  sandbox:
+    name: Register & Serialize example
```

Comment:
```suggestion
    name: Register & Serialize Test Workflows
```

Reply:
Is there a reason we are only serializing "some" (the ones declared in tests) as opposed to everything in core/case studies and integrations ?

Reply:
no we are serializing all but only register them who is declared in tests

Reply:
And why is that?

Reply:
it's not the right flow but we have all examples list in the manifest. We are not leaving anything but ideally we should use one flow for all action